pom.xml

```diff
@@ -862,6 +829,128 @@
   </distributionManagement>
 
   <profiles>
+    <profile>
+      <!--
+       The 'no-jenkins-bom' profile captures the properties & dependencies when _not_ using the Jenkins BOM:
+        https://github.com/jenkinsci/jenkins/blob/master/bom/pom.xml
+       When the BOM is not being used, this profile is adding back the <dependencyManagement/> entries that were
+       moved to the BOM.
+       The profile is enabled by default; if you _are_ using the Jenkins BOM then disable this profile by defining the
+       'use-jenkins-bom' system property.
+      -->
+      <id>no-jenkins-bom</id>
+      <activation>
+        <property>
+          <name>!use-jenkins-bom</name>
+        </property>
+      </activation>
+      <properties>
+        <slf4jVersion>1.7.26</slf4jVersion>        
```

Comment:
This is an update like that in https://github.com/jenkinsci/jenkins/pull/4118. Is there a reason we cannot pick this up from the BOM?

Reply:
this is in the profile which doesn't use the bom, if you see the `use-jenkins-bom` profile it's not set there

Reply:
Missed that, thanks.